content/en/docs/overview/singleton-sdk/Angular Client Introduction/localized-component-mixin.md

```diff
@@ -19,43 +19,118 @@ It also is useful to avoid using the translate pipe in HTML as the Angular linte
 
 #### **Example Code**
 
-![localized-component-mixin-1](https://github.com/zmengjiao/singleton/raw/website/content/en/images/localized-component-mixin/localized-component-mixin-1.png)
+```
+// source.10n.ts
+export const ENGLISH = {
+    "my-key": "Some english for my-key {0} {1}",
+    "my-key-with-html": `
+        <em>Some english in emphasis {0} {1}</em>
+    `
+};
 
+```
 
-![localized-component-mixin-2](https://github.com/zmengjiao/singleton/raw/website/content/en/images/localized-component-mixin/localized-component-mixin-2.png)
+```
+// demo.component.ts
+import { Component } from "@angular/core";
+import { L10nService, LocalizedComponent, Mixin } from "@singleton-i18n/angular-client";
+import { ENGLISH } from "./my-component.l10n.ts";
 
+@Component({
+    selector: "my-component",
+    templateUrl: "./my-component.template.html",
+})
+@Mixin([LocalizedComponent], {
+    L10nKey: "my-component",
+    L10nSourceMap: ENGLISH,
+})
+export class MyComponent implements LocalizedComponent {
+    translate: (key: string, ...args: any[]) => string;
 
-![localized-component-mixin-3](https://github.com/zmengjiao/singleton/raw/website/content/en/images/localized-component-mixin/localized-component-mixin-3.png)
+    constructor(public l10nService: L10nService) {}
+}
 
+```
+
+
+```
+<p>
+    {{translate('my-key', 'argument 1', 'argument 2')}}
+</p>
+
+<div [innerHTML]="translate('my-key-with-html', 'argument 1', 'argument 2')"></div>
+
+```
 
 #### **Collecting/Extracting Source English**
 
 With the English sources isolated to a single type of file, we can easily write a script that collects all the English and dumps it into a JSON file for passing it to the translation team.
 
 The @vmw/ngx-vip library provides a script called collect-mixin-source which will find all files in your project that end in .l10n.ts and pass them through the TypeScript compiler and send them to singleton for translation.
```

Comment:
The Singleton Angular Client library provides a script called collect-mixin-source which will find all files in your project that end in .l10n.ts and pass them through the TypeScript compiler and send them to singleton for translation.